test/features/editing_an_article_test.rb

```diff
@@ -0,0 +1,30 @@
+require "test_helper"
+
+feature "Ability to edit and modify a blog post" do
+  scenario "Clicking edit button brings up blog post" do
+
+    visit articles_path
+    click_link 'New Article'
+    #given
+    fill_in "Title", with: "Code Rails"
+    fill_in "Body", with: "This is how I learned to make web apps"
+
+  #when I submit the form
+  click_on "Create Article"
+
```

Comment:
Agreed. That's my 6AM project tomorrow.  I'll be working on a different
branch and push it ASAP.
On Mar 6, 2015 6:23 PM, "Mike Taylor" <notifications@github.com> wrote:

> In test/features/editing_an_article_test.rb
> <https://github.com/trendwithin/portfolio/pull/2#discussion_r25992205>:
>
> > @@ -0,0 +1,30 @@
> > +require "test_helper"
> > +
> > +feature "Ability to edit and modify a blog post" do
> > +  scenario "Clicking edit button brings up blog post" do
> > +
> > +    visit articles_path
> > +    click_link 'New Article'
> > +    #given
> > +    fill_in "Title", with: "Code Rails"
> > +    fill_in "Body", with: "This is how I learned to make web apps"
> > +
> > +  #when I submit the form
> > +  click_on "Create Article"
> > +
>
> You could DRY up this test by replacing lines 6 through 13 with one line:
>
>  article = Article.create(title: 'Code Rails', body: 'This is how I learned to make web apps')
>
> That change would make this test use the same approach you're using in
> test/features/deleting_an_article_test.rb (see line 6
> <https://github.com/trendwithin/portfolio/blob/articles/test/features/deleting_an_article_test.rb#L6>).
>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/trendwithin/portfolio/pull/2/files#r25992205>.
>